src/org/opendatakit/briefcase/ui/export/components/ExportFormsTableView.java

```diff
@@ -80,6 +83,7 @@
     setFillsViewportHeight(true);
 
     TableRowSorter<ExportFormsTableViewModel> sorter = sortBy(getModel(), FORM_NAME_COL, ASCENDING);
+    sorter.setComparator(SELECTED_CHECKBOX_COL, (Comparator<Boolean>) UI::compareSelectionButton);
```

Comment:
I believe we need to set a couple more comparators for the details and conf button columns as in:
```
sorter.setComparator(OVERRIDE_CONF_COL, (Comparator<ExportConfigurationButton>) ExportConfigurationButton::compare);
sorter.setComparator(DETAIL_BUTTON_COL, (Comparator<DetailsStatusButton>) DetailsStatusButton::compare);

Reply:
See earlier comment on Comparable